src/Umbraco.Web.UI.Client/src/common/services/iconhelper.service.js

```diff
@@ -209,15 +209,11 @@ function iconHelper($http, $q, $sce, $timeout, umbRequestHelper) {
                             ,'Failed to retrieve icon: ' + iconName)
                         .then(icon => {
                             if(icon) {
-                                var trustedIcon = {
-                                    name: icon.Name,
-                                    svgString: $sce.trustAsHtml(icon.SvgString)
-                                };
-                                this._cacheIcon(trustedIcon);
+                                var trustedIcon = this.defineIcon(icon.name, icon.SvgString);
```

Comment:
This leads to errors
```suggestion
                                var trustedIcon = this.defineIcon(icon.Name, icon.SvgString);
```